ribbon-loadbalancer/src/main/java/com/netflix/loadbalancer/AbstractServerList.java

```diff
@@ -49,7 +49,7 @@
             AbstractServerListFilter<T> abstractNIWSServerListFilter = 
                     (AbstractServerListFilter<T>) ClientFactory.instantiateInstanceWithClientConfig(niwsServerListFilterClassName, niwsClientConfig);
             return abstractNIWSServerListFilter;
-        } catch (Throwable e) {
+        } catch (InstantiationException | IllegalAccessException | ClassNotFoundException e) {
```

Comment:
same comment as previous; this is a breaking change in terms of exception handling for no net benefit.